.github/workflows/run-all-tests.yml

```diff
@@ -39,7 +39,6 @@ jobs:
           - os: ubuntu-20.04
             arch: "linux"
             cache: "/home/runner/.cache/bazel-disk"
-            bazel_args: "//launcher/android:jazzer_android"
```

Comment:
Are we still building the Android targets in CI without this?

Reply:
no we wont be building the Android targets without this. This change is correcting the Android build, but does break this CI. Android build wont be able to be built now without the --config=android_arm flag